core/src/main/java/io/questdb/cutlass/http/HttpConnectionContext.java

```diff
@@ -506,8 +506,8 @@ private HttpRequestProcessor checkProcessorValidForRequest(
             if (!multipartProcessor) {
                 if (multipartRequest) {
                     return rejectProcessor.reject(HTTP_NOT_FOUND, "Method (multipart POST) not supported");
-                } else {
-                    return rejectProcessor.reject(HTTP_NOT_FOUND, "Method not supported");
+                } else if (!(processor instanceof HttpContentListener)) {
```

Comment:
we should introduce a concept of POSTRequestProcessor, which both multipart and non-multipart. "http content" is not a standard terminology

Reply:
done, it is called `HttpPostPutProcessor`